extensions/artifacts/org.wso2.developerstudio.codenvy.ext.appfactory/src/main/java/org/wso2/developerstudio/codenvy/ext/appfactory/shared/dto/AppFactoryLoginInfo.java

```diff
@@ -0,0 +1,58 @@
+/*
+* Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.wso2.developerstudio.codenvy.ext.appfactory.shared.dto;
+
+/**
+ * A bean class that keeps login information necessary to authentication related operations
+ */
+public class AppFactoryLoginInfo {
+    private String serverURL;
+    private String userName;
+    private String password;
+    private boolean isAppCloud;
```

Comment:
How does log-in request processing change depending on the fact that the log-in request corresponds to app cloud or not?